demos/layout/layouts.php

```diff
@@ -8,7 +8,7 @@
 $buttons = [
     ['page' => ['layouts_nolayout'],               'title' => 'HTML without layout'],
     ['page' => ['layouts_manual'],                 'title' => 'Manual layout'],
-    ['page' => ['/demos/basic/header', 'layout' => \atk4\ui\Layout\Centered::class], 'title' => 'Centered layout'],
+    ['page' => ['/demos/basic/header', 'layout' => 'Centered'], 'title' => 'Centered layout'],
```

Comment:
This does not work - I think it is some leftover from some not properly done merge... Should be probably solved like any other layout demos - like `layouts_manual` etc.

Reply:
Sure does. The layout is load via $_GET argument:
https://ui.agiletoolkit.org/demos/basic/header.php?layout=Centered

Reply:
You are using absolute URL there - `basic/header` might work

Reply:
![image](https://user-images.githubusercontent.com/2228672/84676162-746e5b80-af2d-11ea-9960-08b9e4621a7e.png)

@ibelar there are still two more absolute paths in the projects to fix

Reply:
these are separate issues. Let merge this and I will check the others laters on.

Reply:
I created an issue for that - https://github.com/atk4/ui/issues/1289 - and merging this one.